gammapy/image/utils.py

```diff
@@ -121,11 +123,20 @@ def disk_correlate(image, radius, mode='constant'):
 
     Parameters
     ----------
-    TODO
+    image : array_like
+        Image to be correlated.
+
+    radius : float
+        Disk radius in pixels.
+
+    mode : {‘reflect’,’constant’,’nearest’,’mirror’, ‘wrap’}, optional
```

Comment:
@mapazarr - There's an error on travis-ci:
https://travis-ci.org/gammapy/gammapy/builds/55911323
https://travis-ci.org/gammapy/gammapy/jobs/55911332#L1411

I think it's because you chose a non-ascii character here ... you should use single-quotes:
```
mode: {'reflect', ...}
```

You should be able to reproduce (and then fix) this issue locally by running
```
python setup.py build_sphinx
```
You should always do this when you work on docstrings or docs and also manually check the generated HTML output via `open docs/_build/html/index.html` (or bookmark that page in your browser).